default_system_message.txt

```diff
@@ -221,6 +221,22 @@ The prompt for the specific refactoring step may request "extra output fields" i
   ```
 ']. ['These should be formatted like the standard fields, above, using second-level Markdown header syntax delimited by single blank lines followed by the literal text (possibly multi-line) for their field values. These field names will be lower-case and may contain underscores.'])
 
+m5_if($md, ['
+  These should be formatted like the standard fields, above, using a second-level Markdown header ## extra_fields and then third-level headers for each key under it. For example:
+
+  ## issues
+  ...
+  ## notes
+  ...
+  ## plan
+  ...
+  ## extra_fields
+  ### clock
+  clk
+  ### reset
+  rst
+']. ['These should be formatted with "extra_fields" as a standard second-level Markdown header field (## extra_fields), followed by individual extra field names using third-level Markdown header syntax (### fieldname) delimited by single blank lines. Field values are literal text (possibly multi-line) that may contain underscores.'])
```

Comment:
I think the only options are "md" and "json", so this case should no longer happen. Make sure the M5 consistently checks for `md` or not `json`.